### PR TITLE
Adding mixed single options and option groups

### PR DESCRIPTION
**Pull request by indacloud**
- changing the render method to handle both choices in groups and the ones without

### CODE REVIEW

File: assets/scripts/src/choices.js [hunk 1]
```diff
@@ -507,7 +507,22 @@ class Choices {
 
           // If we have grouped options
           if (activeGroups.length >= 1 && this.isSearching !== true) {
-            choiceListFragment = this.renderGroups(activeGroups, activeChoices, choiceListFragment);
+            activeChoices.reduce((groupIds, activeChoice) => {
+              const groupId = activeChoice.groupId;
+              // If the choice is not part of a group
+              if (groupId < 0) {
```
Comment (xbilldozer) [inline assets/scripts/src/choices.js]: This works even when the `groupId` is `undefined`? Or is that no longer a case we need to handle?

Comment (indacloud) [reply assets/scripts/src/choices.js]: looks like the groupId is always defined... at least when there is 1+ group

Comment (xbilldozer) [reply assets/scripts/src/choices.js]: https://github.com/kkomaz/Choices/blob/abafb18a450f0e4253f98fa9a9dab24f0f16b223/assets/scripts/src/choices.js#L1021 << this was my concern

Comment (indacloud) [reply assets/scripts/src/choices.js]: ok, I ll add it.. better secure it :)

**Pull request merged**